### ISSUE TITLE
Combobox open arrow has no keyboard focus

### ISSUE THREAD

**Issue by makandre** [IBM/carbon-components-angular#2153]
## Detailed description

> Describe in detail the issue you're having.

When tabbing through a combobox control, the arrow to open/close the dropdown receives focus but does not have focus outline.  This is an accessibility problem

![image](https://user-images.githubusercontent.com/17596240/173657289-5ffab2a3-255e-461f-aac3-4ebd487fe953.png)

> Is this a feature request (new component, new icon), a bug, or a general issue?

Bug

> Is this issue related to a specific component?

Combobox

> What did you expect to happen? What happened instead? What would you like to see changed?

There should be outline on the arrow when it receives focus

> What browser are you working in?

Chrome

> What version of the Carbon Design System are you using?

`"carbon-components-angular": "^4.58.3"`

> What offering/product do you work on? Any pressing ship or release dates we should be aware of?

IBM ADDI

## Steps to reproduce the issue

This can be seen in the storybook https://angular.carbondesignsystem.com/?path=/story/components-combobox--basic

1. Step one
2. Step two
3. Step three
4. etc.

## Additional information

* Screenshots or code
* Notes
